docs/topics/mta-cli-transform.adoc

```diff
@@ -45,6 +45,7 @@ Flags:
   -h, --help            help for openrewrite
   -i, --input string    path to application source code directory
   -l, --list-targets    list all available OpenRewrite recipes
+  -s, --maven-settings string   path to a custom maven settings file to use
```

Comment:
The description for the `--maven-settings` flag is not properly aligned

If the data types for the other flags are not specified, I think it shouldn't be specified for this one either